intg-androidtv/config.py

```diff
@@ -121,7 +127,6 @@ def clear(self) -> None:
         if os.path.exists(self._cfg_file_path):
             os.remove(self._cfg_file_path)
 
-        # FIXME #14 does not work for multi-device support
         pem_file = os.path.join(self._data_path, "androidtv_remote_cert.pem")
```

Comment:
clear() must remove all certificates

Reply:
Ok done : loop over devices to retrieve cert/key file names and remove them. I have not tested it though